quality-assurance/test-code-standards.md

```diff
@@ -1,49 +1,73 @@
 # Test Automation Standards
 
-## C# and Java
+## Test Code
```

Comment:
This doc strays into the same area as the [microservice testing doc](https://github.com/UKHO/docs/blob/main/quality-assurance/microservice-testing.md), i.e. talking about the different types of testing and how much of each you should do.  There's a lot of duplication.  Can you please consolidate into a single doc?